objective.py

```diff
@@ -1,26 +1,73 @@
-import numpy as np
 from benchopt import BaseObjective
+import numpy as np
+from scipy import optimize
```

Comment:
```suggestion
from benchopt import BaseObjective, safe_import_context
with safe_import_context() as import_ctx:
    import numpy as np
    from scipy import optimize
```

Reply:
You need to protect these import to avoid bad interaction with  auto completion.